app/src/main/java/com/hous/hous_aos/ui/rules/today_to_do/tem_dialog/TempMangerAdapter.kt

```diff
@@ -0,0 +1,68 @@
+package com.hous.hous_aos.ui.rules.today_to_do.tem_dialog
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.hous.hous_aos.data.entity.rules.HomieData
+import com.hous.hous_aos.databinding.ItemRulesTmpManagersBinding
+import com.hous.hous_aos.ui.rules.HomieIconType
+
+class TempMangerAdapter(private val setSelectedTmpManager: (Int) -> Unit) :
+    ListAdapter<HomieData, TempMangerAdapter.TmpManagerViewHolder>(tempManagerDiffUtil) {
+    private val homieIconHashMap: HashMap<String, HomieIconType> = hashMapOf(
+        "RED" to HomieIconType.RED,
+        "BLUE" to HomieIconType.BLUE,
+        "GREEN" to HomieIconType.GREEN,
+        "YELLOW" to HomieIconType.YELLOW,
+        "GRAY" to HomieIconType.GRAY,
+        "PURPLE" to HomieIconType.PURPLE,
+    )
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): TmpManagerViewHolder {
+        val binding =
+            ItemRulesTmpManagersBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return TmpManagerViewHolder(binding, homieIconHashMap, setSelectedTmpManager)
+    }
+
+    override fun onBindViewHolder(holder: TmpManagerViewHolder, position: Int) {
+        val currentItem = currentList[position]
+        return holder.onBind(currentItem)
+    }
+
+    class TmpManagerViewHolder(
+        private val binding: ItemRulesTmpManagersBinding,
+        private val homieIconHashMap: HashMap<String, HomieIconType>,
+        private val setSelectedTmpManager: (Int) -> Unit
+    ) :
+        RecyclerView.ViewHolder(binding.root) {
+        fun onBind(data: HomieData) {
+            binding.data = data
+            binding.homieIconType = homieIconHashMap[data.typeColor]
+            binding.ivManagerIcon.setOnClickListener {
+                setSelectedTmpManager(absoluteAdapterPosition)
+                val flag = binding.ivManagerIcon.isSelected
+                binding.ivManagerIcon.isSelected = !flag
+            }
```

Comment:
```suggestion
                setSelectedTmpManager(absoluteAdapterPosition)
                binding.ivManagerIcon.isSelected = !binding.ivManagerIcon.isSelected
            }
```
이렇게도 할 수 있지 않을까요오오

근데 viewmodel에 값을 변경하면, 여기서 isSelected를 바꾸지 않아도 변경 되야 되는 거 아닌가요???

Reply:
맞습니다 센세..